TESTS/mbed_hal/sleep/main.cpp

```diff
@@ -218,13 +227,19 @@ utest::v1::status_t greentea_failure_handler(const Case * const source, const fa
     return STATUS_CONTINUE;
 }
 
+us_timestamp_t disable_stats_read(void)
+{
+    return 0;
+}
+
 utest::v1::status_t greentea_test_setup(const size_t number_of_cases)
 {
     GREENTEA_SETUP(60, "default_auto");
     us_ticker_init();
 #if DEVICE_LPTICKER
     lp_ticker_init();
 #endif
+    mbed_stats_ticker_read = disable_stats_read;
```

Comment:
Rather than disabling the stats ticker, would it be possible the test to gracefully handle this case? This could be done either by decreasing the tolerance as in #7070 or by increasing delay time so the overhead of the stats read is proportionately less.

Reply:
Can be done, delta have to be increased as follows (additional 10-15us to cover worst case: nrf51):
```cpp
/* Used for regular sleep mode, a target should be awake within 10 us. Define us delta value as follows:
 * delta = default 10 us + worst ticker resolution + extra time for code execution + extra time for sleep time logging*/
static const uint32_t sleep_mode_delta_us = (10 + 4 + 5 + 10);
```

Reply:
Looks good for now (I still would expect one day to test 10us period - no logging included - "release")